ampersand-dom-bindings.js

```diff
@@ -5,13 +5,18 @@ var matchesSelector = require('matches-selector');
 var partial = require('lodash.partial');
 var slice = Array.prototype.slice;
 
-function getMatches(el, selector) {
+function getMatches(el, selector, firstOnly) {
     if (selector === '') return [el];
     var matches = [];
-    if (matchesSelector(el, selector)) matches.push(el);
-    return matches.concat(slice.call(el.querySelectorAll(selector)));
+    if(!selector) return matches;
```

Comment:
We already have a test for `''` as a selector above, because `getSelector` guarantees at least that being returned.  I do not believe this `if` is needed, or indeed will ever evaluate to true.